frontend/src/components/Pages/MissionPage/MissionHeader/MissionHeader.tsx

```diff
@@ -9,10 +9,17 @@ import { useLanguageContext, TranslateTextWithContext } from 'components/Context
 import { StatusReason } from '../StatusReason'
 import { MissionRestartButton } from 'components/Pages/FrontPage/MissionOverview/MissionRestartButton'
 
-const HeaderSection = styled.div`
+const HeaderSection = styled(Card)`
```

Comment:
I would prefer if only the title and buttons were sticky. When the status, start date, start time etc. are visible it can be confusing when the mission has a lot of tags.

Reply:
This is a bit difficult since we can't actually achieve this without moving the title out of the header. The CSS sticky feature only works within the parent component. To have something stay the same position on the screen regardless of its context we need to use "fixed" positioning which does not allow us to only start moving it when we scroll down a certain amount